templates/resources/application.md.tmpl

```diff
@@ -0,0 +1,36 @@
+---
+page_title: "{{.Name}} {{.Type}} - {{.ProviderName}}"
+subcategory: ""
+description: |-
+{{ .Description | plainmarkdown | trimspace | prefixlines "  " }}
+---
+
+# {{.Name}} ({{.Type}})
+
+{{ .Description | plainmarkdown | trimspace | prefixlines "  " }}
+
+## Example Usage
+
+
+### Application deployment into Cluster Group.
+
+```hcl
+resource "spectrocloud_application" "application" {
+  name                    = "app-beru-whitesun-lars"
+  application_profile_uid = "6345f67d784e4d30683b9987" # or data.spectrocloud_application_profile.id
```

Comment:
```suggestion
  application_profile_uid = data.spectrocloud_application_profile.id
```
We should add the data resource above and use that as an example versus hard coding. That way we share best practices 😄